trtools/utils/mergeutils.py

```diff
@@ -248,13 +248,20 @@ def GetMinRecords(record_list: List[Optional[trh.TRRecord]], chroms: List[str])
     return [CheckPos(r, chroms[min_chrom], min_pos) for r in record_list]
 
 
+def GetCoordinatesOfRecord(record: Optional[trh.TRRecord]) -> Tuple[int, int]:
+    if record is None:
+        return np.inf, np.inf
 
-def GetMinHarmonizedRecords(record_list: List[Optional[trh.TRRecord]], chroms: List[str]) -> List[bool]:
-    r"""Check if each record is next up in sort order
+    return record.pos, record.pos + record.ref_allele_length * len(record.motif)
```

Comment:
I'm thinking that we should add a field called 'end_pos' to the TRRecord that has this number in it `record.pos + record.ref_allele_length * len(record.motif)` and then just access that here. Do you agree that field may be useful in more than just this context?

Reply:
Yes, I agree. I've also noticed that this: `record.pos + record.ref_allele_length * len(record.motif)` is off by one, so I added end position attribute to TRRecord and fixed this problem

Reply:
Thanks